### PR TITLE
MUMMNG-1734 : Add widget creator to web

### PR DESCRIPTION
**Pull request by timlevett**
This is the first step in splitting out my-tools home into its own app. This also upgrades widget creator to use the latest widget code. They are one in the same now. So now we can do more than creating custom widgets with widget creator. We can demo some of the templates. Picked a couple high usage ones.

### Widget creator enhancements
+ Added a `widgetConfig` section for widgets with config
+ Added `widgetURL`, only used by RSS type for now
+ Added RSS, LOL, and Search with links types to widget creator

### Custom
http://goo.gl/ET9xA7
http://goo.gl/dXvqqS

### RSS
http://goo.gl/CCJ1OQ

### LOL
http://goo.gl/QoKJfD

### Search with Links
http://goo.gl/jd7ssk
http://goo.gl/4ji3L9

### CODE REVIEW

**Comment by keirserrie** [comment]
very cool :+1:

File: angularjs-portal-home/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -66,24 +66,24 @@
...
     <filter-mapping>
        <filter-name>ExpiresFilter</filter-name>
        <url-pattern>/*</url-pattern>
        <dispatcher>REQUEST</dispatcher>
     </filter-mapping>
-    
-    
+
+
     <!-- Keep this in sync with angular-page.js -->
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/WEB-INF/web.xml]: What's `angular-page.js`, and does this changeset keep these `<servlet-mapping>`s in sync with it?

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/WEB-INF/web.xml]: Not sure, probably copied in long ago when this started.

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/WEB-INF/web.xml]: As you can see, i didn't add/modify that line during this change ;)

**Pull request merged**